Products/PleiadesEntity/browser/attestations.py

```diff
@@ -242,3 +252,78 @@ def rows(self, names):
             ]
             output.append(u"\n".join(innerHTML))
         return output
+
+
+class ConnectionsTable(ChildrenTable):
+    """table of connections and associated information for plone views
+    """
+
+    def accessor(self):
+        return self.context.getSubConnections()
+
+    def snippet(self, ob):
+        return unicode(self.referenced(ob).Title(), "utf-8")
+
+    @view.memoize
+    def referenced(self, ob):
+        return ob.getConnection()
+
+    def postfix(self, ob):
+        acert = ob.getAssociationCertainty()
+        timespan = TimeSpanWrapper(ob).snippet
+        if timespan.strip() == '':
+            timespan = None
+        elif timespan.strip() == 'AD 1700 - Present':
+            timespan = 'modern'
+        if timespan:
+            annotation = u'(%s)' % timespan
+        else:
+            annotation = None
+        if acert == 'less-certain':
+            return [u'?', u' %s?' % annotation][annotation is not None]
+        elif acert == 'uncertain':
+            return [u'??', u' %s??' % annotation][annotation is not None]
+        else:
+            return [u'', u' %s' % annotation][annotation is not None]
```

Comment:
If this is the same logic as for the other tables, maybe we should factor it out to avoid duplication.